api/src/main/resources/marquez/db/migration/V72__add_current_lineage_metrics.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS current_hour_lineage_metrics
```

Comment:
Do we need a table, or can we create a view `lineage_events_by_current_hour_view`, `lineage_events_by_current_day_view `?

Reply:
So the idea of this current day and hour table is to not be based off another table, and have it get wiped by dw on the corresponding hour and day intervals... If we wanted to do a view I guess we would base off of `lineage_events` but that is ostensibly the largest table in the db.